backend/cmd/headlamp.go

```diff
@@ -54,24 +59,74 @@ func (h spaHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	path = strings.TrimLeft(path, h.baseURL)
+
 	// prepend the path with the path to the static directory
 	path = filepath.Join(h.staticPath, path)
 
 	// check whether a file exists at the given path
 	_, err = os.Stat(path)
 	if os.IsNotExist(err) {
 		// file does not exist, serve index.html
-		http.ServeFile(w, r, filepath.Join(h.staticPath, h.indexPath))
+		staticPath := filepath.Join(h.staticPath, h.indexPath)
+		http.ServeFile(w, r, staticPath)
+
 		return
 	} else if err != nil {
 		// if we got an error (that wasn't that the file doesn't exist) stating the
 		// file, return a 500 internal server error and stop
 		http.Error(w, err.Error(), http.StatusInternalServerError)
+
```

Comment:
Shouldn't be here.

Reply:
The extra line? I think the golang lint tool asked for it.